lib/impure/db_sqlite.nim

```diff
@@ -713,11 +720,70 @@ proc setEncoding*(connection: DbConn, encoding: string): bool {.
   exec(connection, sql"PRAGMA encoding = ?", [encoding])
   result = connection.getValue(sql"PRAGMA encoding") == encoding
 
-proc finalize*(sqlPrepared:SqlPrepared){.discardable.} = 
+proc finalize*(sqlPrepared:SqlPrepared) {.discardable, since: (1, 3).} = 
   discard finalize(sqlPrepared.PStmt)
 
+template dbBindParamError*(paramIdx: int, val: untyped) =
+  ## Raises a `DbError` exception.
+  var e: ref DbError
+  new(e)
+  e.msg = "error binding param in position " & $paramIdx 
+  raise e
+
+proc bindParam*(ps: SqlPrepared, paramIdx: int, val: int32) {.since: (1, 3).} =
+  ## Binds a int32  to the specified paramIndex.
+  if bind_int(ps.PStmt, paramIdx.int32, val) != SQLITE_OK:
+    dbBindParamError(paramIdx, val)
+
+
+proc bindParam*(ps: SqlPrepared, paramIdx: int, val: int) {.since: (1, 3).} =
+  ## Binds a int  to the specified paramIndex.
+  when sizeof(int) == 8:
+    if bind_int64(ps.PStmt, paramIdx.int32, val.int64) != SQLITE_OK:
+      dbBindParamError(paramIdx, val)
+  else:
+    if bind_int(ps.PStmt, paramIdx.int32, val.int32) != SQLITE_OK:
+      dbBindParamError(paramIdx, val)
```

Comment:
Instead of duplicating code, move this proc below the `int64` overload, then call each of the overloads depending on the size of `int`